src/xenia/cpu/backend/x64/x64_backend.cc

```diff
@@ -521,10 +564,18 @@ void X64ThunkEmitter::EmitLoadVolatileRegs() {
   mov(r9, qword[rsp + offsetof(StackLayout::Thunk, r[4])]);
   mov(r10, qword[rsp + offsetof(StackLayout::Thunk, r[5])]);
   mov(r11, qword[rsp + offsetof(StackLayout::Thunk, r[6])]);
+#endif
 }
 
 void X64ThunkEmitter::EmitSaveNonvolatileRegs() {
-  // Preserve nonvolatile registers.
+#if XE_PLATFORM_LINUX
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[0])], rbx);
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[1])], rbp);
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[2])], r12);
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[3])], r13);
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[4])], r14);
+  mov(qword[rsp + offsetof(StackLayout::Thunk, r[5])], r15);
```

Comment:
N.B: SysV does not have nonvolatile XMM registers.

Reply:
What is the suggested way of resolving this?

Reply:
Sorry - I think this was a note for myself. I might suggest you just drop a comment here and state what I said, just so nobody goes "wut, why aren't xmm registers being saved"